validator/controller.go

```diff
@@ -148,6 +152,22 @@ func (c *controller) GetValidatorsPubKeys() [][]byte {
 	return pubKeys
 }
 
+// GetValidatorsPubKeys returns a list of all the active validators indices and fetch indices for missing once (could be first time attesting or non active once)
```

Comment:
@nivBlox all active validators for the node or all eth2 active validators?

Reply:
all active shares.